[INJIWEB-1149]: Fixed Borders Visiblity

kamlesh012 opened the pull request.

Gurpreet41082 commented on inji-web/src/modals/ModalWrapper.tsx:
```diff
@@ -1,19 +1,19 @@
-import React from "react";
+import React from 'react';
 import {ModalWrapperProps} from "../types/components";
```
@kamlesh012 check if any UI test case needs to be updated.

PuBHARGAVI replied:
@Gurpreet41082 In Modal wrapper test case file we are checking if the div elements are rendered in the UI or not based on their test id and since test id's are not changed the test cases are working as expected. So to keep track of styling changes we can add a new snapshot test case if we want @kamlesh012

kamlesh012 replied:
Ok, looking into it.

kamlesh012 replied:
Added Snapshot Test Case in ModalWrapper.test.tsx

PuBHARGAVI reviewed (commented):
Verified the changes in the UI and it LGTM

kamlesh012 commented:
Increased the Margins on the sides of Share Visibility Modal so that even on smaller screens the margins outside of MODAL are visible.
Fixed a scrolling bug.Now background won't get scrolled with Share Visibility Modal is open.
Fixed Issuer Logo Resize & Disapper issue on Mobile Devices.
Fixed Text Overflowing out of Container & Text Overlap with Radio Button, in smaller resolutions, on Share Visibility page when selecting language as Portugues.
